README.md

```diff
@@ -48,6 +49,7 @@ The [`examples`](./examples) directory contains examples for using the container
 | Vertex AI | [examples/vertex-ai/notebooks/trl-full-sft-fine-tuning-on-vertex-ai](./examples/vertex-ai/notebooks/trl-full-sft-fine-tuning-on-vertex-ai) | Fine-tune Mistral 7B v0.3 with PyTorch Training DLC using SFT on Vertex AI  |
 | GKE       | [examples/gke/trl-full-fine-tuning](./examples/gke/trl-full-fine-tuning)                                                                   | Fine-tune Gemma 2B with PyTorch Training DLC using SFT on GKE               |
 | GKE       | [examples/gke/trl-lora-fine-tuning](./examples/gke/trl-lora-fine-tuning)                                                                   | Fine-tune Mistral 7B v0.3 with PyTorch Training DLC using SFT + LoRA on GKE |
+| TPU       | [gemma-tuning](https://github.com/huggingface/optimum-tpu/blob/main/examples/language-modeling/gemma_tuning.ipynb)                    | Fine-tune Gemma 2B with PyTorch Training DLC using LoRA                     |
```

Comment:
Question about this. We have a training example but it is in optimum TPU do we want to link to it from this Readme or is it better to not include it. I don't think copying the examples to this repo is really the best. Also the example is embedded in the container so it is easily accessible to the user

Reply:
Not sure we want to ship the examples within the containers here, and ideally here we should link the examples directly relying on the DLCs, but we can always include a reference to all the `optimum-tpu` examples, right?

Reply:
Ok i removed it then as either way we have a note linking to the rest of the `optimum-tpu` examples 

https://github.com/huggingface/Google-Cloud-Containers/blob/1474a747ace75cbbb2a440028fbfaf7caabf050a/containers/pytorch/training/README.md?plain=1#L65